api/notd/model.py

```diff
@@ -20,6 +21,18 @@ class RetrievedTokenTransfer:
     blockHash: str
     blockDate: datetime.datetime
 
+@dataclasses.dataclass
+class TokenMetadata:
+    createdDate: datetime.datetime
+    updatedDate: datetime.datetime
+    registryAddress: str
+    tokenId: int
+    metdataUrl: str
+    imageUrl: int
```

Comment:
and here many of these are non-optional but are nullable in the table definition below. please check everything is consistent here.